packages/devextreme/artifacts/npm/devextreme/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I remember this file was committed some time ago, then `.gitignore`'ed, is it intentionally again here?

Reply:
Yes. Path added to .gitignore when this task was in progress, so that file added to index
I removed this package.json